core/GlobalState.h

```diff
@@ -223,6 +223,10 @@ class GlobalState final {
     // Contains a string to be used as the base of the error URL.
     // The error code is appended to this string.
     std::string errorUrlBase;
+
+    // If 'true', issue an error for pre-Ruby 2.7-style keyword hash argument (without '**').
+    bool errorKwargHashWithoutSplat = false;
```

Comment:
I think we will likely want to add multiple Ruby 3 keywords-related errors, and I don't know that it makes sense to add options for each one of them, given that you can already `--error-black-list` individual error codes if you don't want every kind of keyword args error.

Reply:
@jez So something like `--error-obsolete-kwargs`, which would enable this new error but possibly _also_ enable future kwarg-related errors?

Reply:
Yes, I would have called the flag `--ruby3-keyword-args` and made it emit errors for code known to cause errors in Ruby 3.0 (i.e., deprecation warnings in Ruby 2.7)